src/simularium/VisData.ts

```diff
@@ -259,32 +179,38 @@ class VisData {
                 this.frameToWaitFor = 0;
             }
         }
-
+        const parsedMsg: CachedFrame = parseVisDataMessage(visDataMsg);
+        if (
+            this.frameCache.cacheSizeLimited &&
+            parsedMsg.size > this.frameCache.maxSize
+        ) {
+            this.onError(
+                new FrontEndError(
+                    `Frame size exceeds cache size: ${parsedMsg.size} > ${this.frameCache.maxSize}`,
+                    ErrorLevel.ERROR
+                )
```

Comment:
Providing error handling callback to VisData for basically just these scenarios (one for binary, one for JSON), so that the user sees a pleasant warning modal and not an uncaught runtime error if their cache is too small for their data.

Reply:
At this point the default is unlimited.

There was discussion of setting default limits based on different browsers, but I deferred thinking that through. This will only be reached if a default cache size has been manually set.